applications/FluidDynamicsApplication/python_registry_lists.py

```diff
@@ -5,4 +5,10 @@
 python_processes_to_be_registered = [
     "apply_boussinesq_force_process.ApplyBoussinesqForceProcess",
     "apply_inlet_process.ApplyInletProcess"
-]
\ No newline at end of file
+]
+
+python_stages_to_be_registered = [
+    "fluid_dynamics_analysis.FluidDynamicsAnalysis"
+]
+
+python_multistage_orchestrators_to_be_registered = []
```

Comment:
We might check if the attribute is in the module but I have the feeling that having them empty will ease developers in populating the registry. If we put it empty people only needs to fill it with their stuff. Removing it would require them to browse through the code to find which is the list name the registry is expecting.